wai-app-static/WaiAppStatic/Storage/Filesystem.hs

```diff
@@ -120,8 +119,8 @@ webAppLookup hashFunc prefix pieces =
 -- exists.
 hashFile :: FilePath -> IO ByteString
 hashFile fp = do
-    h <- Crypto.Hash.Conduit.hashFile fp
-    return $ B64.encode $ toBytes (h :: Digest MD5)
+    f <- BL.readFile fp
```

Comment:
Lazy ByteString's `readFile` closes the FD ("The Handle will be held open until EOF is encountered"). I've tried `withBinaryFile`, it results in `hGetBufSome: illegal operation (handle is closed)`